From b6d7a3f37565530488098b7bf47cf3211b57b352 Mon Sep 17 00:00:00 2001
From: Chengwei Yang <chengwei.yang@intel.com>
Date: Wed, 26 Jun 2013 15:40:12 +0800
Subject: [PATCH 2/3] FreeBSD: explicit include signal.h to fix build failure

In Linux envrionment, signal.h included by sys/wait.h, however, this
isn't the case in FreeBSD. So explicit include it to fix build failure.

Bug: https://bugs.freedesktop.org/show_bug.cgi?id=66197
Signed-off-by: Chengwei Yang <chengwei.yang@intel.com>
Reviewed-by: Simon McVittie <simon.mcvittie@collabora.co.uk>
Origin: upstream, commit:9665b8ecd84c650804945104d1ffe4f06cce199f
---
 tools/dbus-run-session.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/dbus-run-session.c b/tools/dbus-run-session.c
index 4f0b32b..4f7243f 100644
--- a/tools/dbus-run-session.c
+++ b/tools/dbus-run-session.c
@@ -34,6 +34,7 @@
 
 #include <sys/types.h>
 #include <sys/wait.h>
+#include <signal.h>
 
 #define MAX_ADDR_LEN 512
 #define PIPE_READ_END  0
-- 
1.8.4.rc3

